packages/@apphosting/common/src/index.ts

```diff
@@ -31,26 +31,31 @@ interface ServerConfig {
   maxInstances?: number;
 }
 
-// Additonal fields needed for adapter identification
+// Additonal fields needed for identifying the framework and adapter being used
 interface Metadata {
-  // Name of the adapter (this should be the npm package name)
-  adapterNpmPackageName: string;
-  // Version of the adapter
+  // Name of the adapter (this should be the official package name) e.g. "@apphosting/adapter-nextjs"
+  adapterPackageName: string;
+  // Version of the adapter, e.g. "18.0.1"
   adapterVersion?: string;
-  // Name of the framework that is being supported
+  // Name of the framework that is being supported, e.g. "angular"
   framework: string;
-  // Version of the framework that is being supported
+  // Version of the framework that is being supported, e.g. "18.0.1"
   frameworkVersion?: string;
 }
 
-// Configs necessary to define environment variables
+// Represents a single environment variable.
 interface EnvVarConfig {
   // Name of the variable
   variable: string;
   // Value associated with the variable
   value: string;
-  // Where the variable will be available, for now this will always be RUNTIME
-  availability: "RUNTIME"[];
+  // Where the variable will be available, for now only RUNTIME is supported
+  availability: Availabilities.Runtime[];
+}
+
+// Represents where environment variables are available from
```

Comment:
comment nit: this is a bit confusing, how about "Represents where environment variables are made available."

Reply:
Done